packages/testing/src/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -307,6 +308,12 @@ protected function doTestFileInfo(SmartFileInfo $fixtureFileInfo, array $extraFi
             return;
         }
 
+        $rectorClass = $this->getRectorClass();
+        if (self::$container->has($rectorClass)) {
+            $rectorObject = self::$container->get($rectorClass);
+            $this->assertInstanceOf(RuleDefinition::class, $rectorObject->getRuleDefinition());
+        }
+
```

Comment:
This test does not bring much value. This is tested by PHP itself as there is `RuleDefinition` return type

Reply:
ok 👍